lua/grpc.lua

```diff
@@ -252,10 +271,11 @@ else -- hook env
     end
   end
 
+  local skipFrames = math.ceil(interval / 0.016) -- 0.016 = 16ms = 1 frame at 60fps
   local frame = 0
   function GRPC.onSimulationFrame()
     frame = frame + 1
-    if frame >= 10 then
+    if frame >= skipFrames then
```

Comment:
Since we are currently only able to set settings for the mission scripting env, the hook env will still always fall back to the default 600. Probably something I'll postpone for a follow-up PR.

- [ ] Create a note somewhere about the followup